frontend/src/features/projects/common/projectConverter.ts

```diff
@@ -168,36 +177,74 @@ const isParcelOrSubdivision = (property: IProperty) =>
 /** create api evaluation objects based on flat app evaluation structure */
 const getApiEvaluations = (property: IProperty): IEvaluation[] => {
   const evaluations: IEvaluation[] = [];
-  evaluations.push({
-    parcelId: isParcelOrSubdivision(property) ? property.id : undefined,
-    buildingId: property.propertyTypeId === PropertyTypes.BUILDING ? property.id : undefined,
-    value: isParcelOrSubdivision(property)
-      ? property.assessedLand || 0
-      : property.assessedBuilding || 0,
-    date: isParcelOrSubdivision(property)
-      ? property.assessedLandDate ?? formatDate(new Date())
-      : property.assessedBuildingDate ?? formatDate(new Date()),
-    rowVersion: property.assessedLandRowVersion,
-    key: EvaluationKeys.Assessed,
-    firm: property.assessedLandFirm ?? '',
-  });
-  evaluations.push({
-    parcelId: isParcelOrSubdivision(property) ? property.id : undefined,
-    buildingId: property.propertyTypeId === PropertyTypes.BUILDING ? property.id : undefined,
-    value: isParcelOrSubdivision(property)
-      ? property.assessedLand || 0
-      : property.assessedBuilding || 0,
-    date: isParcelOrSubdivision(property)
-      ? property.assessedLandDate ?? formatDate(new Date())
-      : property.assessedBuildingDate ?? formatDate(new Date()),
-    rowVersion: property.assessedLandRowVersion,
-    key: EvaluationKeys.Improvements,
-    firm: property.assessedLandFirm ?? '',
-  });
+  if (isParcelOrSubdivision(property)) {
```

Comment:
do NOT create new evaluations by default, as there may not be any values provided at the property level.